examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -0,0 +1,698 @@
+{
```

Comment:
<p>Typo "<a href="https://github.com/NVIDIA-Merlin/NVTabular/" target="_blank">NVTabular</a>)" - omit )</p><p> </p><p>For the diagram: What feeds into the first arrow? and what does the last arrow produce?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I added a comment, that we are interested only in the subsequence. </p>